graphgrid_sdk/ggcore/sdk_messages.py

```diff
@@ -312,3 +312,13 @@ def __init__(self, generic_response: GenericResponse):
             self.dag_run_id = loaded.get('dag_run_id')
             self.logical_date = loaded.get('logical_date')
             self.state = loaded.get('state')
+
+
+class GetActiveModelResponse(SdkServiceResponse):
+    """Define class representing a get active model api call response."""
+    task: str
+
+    def __init__(self, generic_response: GenericResponse):
+        super().__init__(generic_response)
+
+        self.task = generic_response.response
```

Comment:
The fields of this model class don't line up with the returned java response `ModelMetrics`. That class has `modelName` and `trainedModelData` fields, but this only has `task`. 

Right now this loads the entire response into this `task` field as a string. I don't think this is correct. 

I think you want to match the `modelName` and `trainedModelData` fields from the java side, and then use a similar approach to our other response objects and extract the response string into a dict with `json.loads(generic_response.response)`. From there you can populate the `modelName` and `trainedModelData` fields.